tstest/integration/integration.go

```diff
@@ -98,3 +110,48 @@ func exe() string {
 	}
 	return ""
 }
+
+func RunDERPAndStun(t testing.TB, logf logger.Logf, ipAddress string) (derpMap *tailcfg.DERPMap, cleanup func()) {
```

Comment:
exported so docs.

in particular, describe ipAddress

Reply:
also, STUN is all caps per https://github.com/golang/go/wiki/CodeReviewComments#initialisms